hazelcast/src/main/java/com/hazelcast/util/PhoneHome.java

```diff
@@ -72,7 +68,7 @@
     private static final int H_INTERVAL = 300;
     private static final int J_INTERVAL = 600;
 
-    private static final String BASE_PHONE_HOME_URL = "http://phonehome.hazelcast.com/ping";
+    private static final String BASE_PHONE_HOME_URL = "http://127.0.0.1:8080/ping";
```

Comment:
forgot this?

Reply:
yes it is typo , fixed 👍 thanks for pointing out